doc/source/serve/production-guide/rest-api.md

```diff
@@ -400,10 +136,6 @@ deployments:
 ...
 ```
 
-(serve-in-production-status-command)=
-
-### `serve status`
-
 `serve status` gets your Serve application's current status. It's divided into two parts: the `app_status` and the `deployment_statuses`.
 
 The `app_status` contains three fields:
```

Comment:
Could you add [`"NOT_STARTED"`](https://github.com/ray-project/ray/blob/4ab97399cda80be1e146946c43d6fb2926248b28/python/ray/serve/_private/common.py#L32) as one of the potential `app_status` `statuses`? When a Ray cluster is running without a Serve app, it's `"NOT_STARTED"`.

Reply:
done